docs/developers/api.md

```diff
@@ -0,0 +1,223 @@
+## Table of Contents
+
+  - [Overview](#overview)
+  - [Search](#search)
+
+## Overview
+
+OpenTrials API manages the schema of our API `database` and search indexes. It also 
+exposes the data in RESTful enpoints.
+
+Resources:
+- an up-to-date documentation of the API endpoints is available [here](https://api.opentrials.net/v1/docs/).
+- more information on the schema of the API `database` in [this section](data-model.md).
+- more techical details (stack, how to set up) [here](https://github.com/opentrials/api/blob/master/README.md)
+
+## Search
+
+Our search is powered by [ElasticSearch](https://www.elastic.co/guide/en/elasticsearch/reference/2.0/).
+
+The general format of a `/search` request is:
+
+```
+/v1/search?q=your_query_here
+```
+
+You can pass as value to the `q` parameter any query on indexed attributes that complies with the
+[ElasticSearch Query String syntax](https://www.elastic.co/guide/en/elasticsearch/reference/2.0/query-dsl-query-string-query.html).
+To see the available attributes for each index check the lists below.
+
+A `GET` request on `/search` routes without `q` parameter will return all results.
+
+Search results are paginated but you can customize the number of results per page.
```

Comment:
That might change and it is available in the Swagger docs which will also reflect any changes so I don't think it's worth including it here.